server.js

```diff
@@ -27,8 +27,7 @@ app.use(express.static('public'));
 
 // Routes
 // =============================================================
-require('./routes/html-routes.js')(app);
-// require("./routes/author-api-routes.js")(app);
+// require('./routes/html-routes.js')(app);
```

Comment:
Please uncomment this line. Seems to have been removed for some reason and prevents the homepage from displaying.